allennlp/data/iterators/adaptive_iterator.py

```diff
@@ -131,3 +132,61 @@ def _adaptive_grouping(self, dataset: Dataset):
             logger.debug("Batch size: %d; padding: %s", len(current_batch), padding_lengths)
         batches.append(current_batch)
         return batches
+
+    @classmethod
+    def from_params(cls, params: Params) -> 'AdaptiveIterator':
+        """
+        Parameters
```

Comment:
I don't think there's any need to copy the documentation for `from_params` - we've only done this in a few cases where more complicated things than just `Params` unpacking happens, such as [here](https://github.com/allenai/allennlp/blob/master/allennlp/training/initializers.py#L62).